api/src/main/java/com/cloud/storage/snapshot/SnapshotApiService.java

```diff
@@ -100,6 +100,13 @@
      */
     Snapshot createSnapshot(Long volumeId, Long policyId, Long snapshotId, Account snapshotOwner);
 
+    /**
+     * Archives a snpashot from primary storage to secondary storage.
```

Comment:
Minor: Spelling issue on what should be the word "snapshot".

Reply:
done